test/index.js

```diff
@@ -474,4 +475,156 @@ describe('HauteCouture', () => {
         });
     });
 
+    it('allows options to be optional', (done, onCleanup) => {
+
+        onCleanup((next) => {
+
+            reset();
+            return next();
+        });
+
+        const server = new Hapi.Server();
+        server.connection();
+
```

Comment:
"Shadowing" is when one a scope declares a variable of the same name as one that exists in an upper scope.  In this case like 489 redeclares `server`.  In the plugin just use `srv` rather than `server`.